### PR TITLE
fix: bump static analysis level

### PR DESCRIPTION
**Pull request by JoshuaLicense**
## Description

Bump the static analysis level.

## Before submitting (or marking as "ready for review")

- [x] Does the pull request title follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) specification?
- [x] Have you performed a self-review of the code
- [x] Have you have added tests that prove the fix or feature is effective and working
- [x] Did you make sure to update any documentation relating to this change?

### CODE REVIEW

File: psalm.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <psalm
-  errorLevel="7"
+  errorLevel="5"
```
Comment (Bal2018) [inline psalm.xml:3]: Why have you lowered the error level?

Comment (JoshuaLicense) [reply psalm.xml:3]: Different tools use different ways to define strictness. In this case, Psalm uses lower levels to increase strictness.

https://psalm.dev/docs/running_psalm/error_levels/

File: test/Functional/ModuleTest.php [hunk 2]
```diff
@@ -60,8 +63,8 @@ public function testProcessParameters(): void
 
         $config = $application->getConfig();
 
-        $this->assertEquals('secret', $config['secret']);
-        $this->assertEquals('parameter', $config['parameter']);
+        $this->assertEquals('secret', $config['secret'] ?? null);
+        $this->assertEquals('parameter', $config['parameter'] ?? null);
```
Comment (Bal2018) [inline test/Functional/ModuleTest.php:67]: Will we ever have it where both secret and parameter store are null ???

Comment (JoshuaLicense) [reply test/Functional/ModuleTest.php:67]: These are unit tests so if they are not set then the test will fail - null is an impossible expected value in this test.

In the source code, these things will always be a string:

https://docs.aws.amazon.com/aws-sdk-php/v3/api/api-secretsmanager-2017-10-17.html#getsecretvalue

https://docs.aws.amazon.com/cli/latest/reference/ssm/get-parameters-by-path.html#output

**Review by Bal2018 — APPROVED** [review]

**Pull request merged**